src/toml/toml_lexer.zig

```diff
@@ -139,11 +139,16 @@ pub const Lexer = struct {
     }
 
     inline fn nextCodepointSlice(it: *Lexer) []const u8 {
+        if (it.current >= it.source.contents.len) return "";
```

Comment:
We should add padding bytes instead imo

Reply:
This will probably be branch-predicted perfectly anyway - I suggest we keep this change to ensure correctness.